src/main/java/medicaldoctor/controllers/LabWorkerDashboardController.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package medicaldoctor.controllers;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.fxml.Initializable;
+
+/**
+ *
+ * @author Salmanhussain
+ */
+public class LabWorkerDashboardController implements Initializable, ParentController{
```

Comment:
we have 2 types of lab workers

Reply:
We have only had LabWorker.fxml that's why I thought we only had one.

Reply:
fair. i guess lets add separating these once we are working on lab stuff